cmd/validate_image_test.go

```diff
@@ -22,23 +22,29 @@ import (
 	"errors"
 	"testing"
 
-	"github.com/hacbs-contract/ec-cli/internal/policy"
 	"github.com/open-policy-agent/conftest/output"
 	appstudioshared "github.com/redhat-appstudio/managed-gitops/appstudio-shared/apis/appstudio.redhat.com/v1alpha1"
-	"github.com/sigstore/cosign/pkg/oci"
 	"github.com/stretchr/testify/assert"
+
+	output2 "github.com/hacbs-contract/ec-cli/internal/output"
```

Comment:
Difficult to see what's output vs output2

```suggestion
	"github.com/hacbs-contract/ec-cli/internal/output"
```